From 6d2934587ba5268816ab9ec02cb1d81487e49fa8 Mon Sep 17 00:00:00 2001
From: Evan Layton <evan.layton@nexenta.com>
Date: Wed, 22 Aug 2018 11:46:01 -0600
Subject: [PATCH] NEX-18312 Max number of nfsd threads is set too high, thus
 affecting NFS performance Reviewed by: Rick McNeal <rick.mcneal@nexenta.com>
 Reviewed by: Rob Gittins <rob.gittins@nexenta.com>

---
 usr/src/cmd/fs.d/nfs/svc/server.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/cmd/fs.d/nfs/svc/server.xml b/usr/src/cmd/fs.d/nfs/svc/server.xml
index 87b6a1ed85..2d9c627f1e 100644
--- a/usr/src/cmd/fs.d/nfs/svc/server.xml
+++ b/usr/src/cmd/fs.d/nfs/svc/server.xml
@@ -176,7 +176,7 @@
 	   <propval name='server_delegation' type='astring' value='on'/>
 	   <propval name='server_versmax' type='integer' value='4'/>
 	   <propval name='server_versmin' type='integer' value='2'/>
-	   <propval name='servers' type='integer' value='1024'/>
+	   <propval name='servers' type='integer' value='256'/>
 	   <propval name='mountd_listen_backlog' type='integer' value='64'/>
 	   <propval name='mountd_max_threads' type='integer' value='16'/>
 	   <propval name='mountd_port' type='integer' value='0'/>
-- 
2.17.2 (Apple Git-113)

